www/js/lib/abstractFilesystemAccess.js

```diff
@@ -234,7 +235,7 @@ function loadPreviousZimFile () {
     // It's a bit hacky but it works and I am not sure if there is any other way ATM
     setTimeout(() => {
         if (window.params.isFileSystemApiSupported || window.params.isWebkitDirApiSupported) {
-            const filenames = localStorage.getItem('zimFilenames');
+            const filenames = settingsStore.getItem('zimFilenames');
             if (filenames) updateZimDropdownOptions(filenames.split('|'), '');
         }
     }, 200);
```

Comment:
This is just a note to self, not part of this PR, but this timeout may well be the problem with inconsistency in loading the contents of archives. It introduces a race condition, and needs looking at as a separate issue.